test/basic.tests.js

```diff
@@ -1092,8 +1100,11 @@ contract("LiquidityBridgeContract", async (accounts) => {
       1000
     );
 
+    const quoteHash = await instance.hashPegoutQuote(quote);
+    const signature = await web3.eth.sign(quoteHash, liquidityProviderRskAddress);
     const tx = instance.depositPegout(
       quote,
+      signature,
       { value: 500 }
```

Comment:
and what is 500?

Reply:
is the value to deposit, I will change to a variable instead